src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,66 @@
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+
+import { ErrorType } from '../../enums/ErrorType';
+
+type Props = {
+  errorType: ErrorType;
+  onCloseNotification: () => void;
+};
+
+export const ErrorNotification: React.FC<Props> = React.memo(
```

Comment:
do you really need the memoization here? sometimes it can cause more issues than bring positive outcomes